lib/minfraud/assessments.rb

```diff
@@ -129,18 +136,22 @@ def score
     private
 
     def perform_request(endpoint)
-      raw = request.perform(
-        verb:     :post,
-        endpoint: endpoint.to_s,
-        body:     request_body,
-      )
+      response = nil
+      body     = nil
+      Minfraud.connection_pool.with do |client|
```

Comment:
Are there any thread-safety differences with this vs Faraday?

Reply:
I think it is safe to use in the same way as before. I followed the same pattern as in GeoIP2-ruby and I believe I looked into it at the time. There's comments to that effect there.